usefulUtilities/airAndWhiteboardDraw/README.md

```diff
@@ -0,0 +1,30 @@
+# Vending Machine
```

Comment:
Fix readme to be accurate

Reply:
related to that - could you please make a list of all of the files you wrote as a part of this project, where they're supposed to be attached, and what those files' purposes are?

Reply:
I added this to the readme now!